File: nginx/MASZ/src/app/models/GuildConfig.ts
export interface GuildConfig {
    id: number;
    guildId: string;
    modRoles: string[];
    adminRoles: string[];
    mutedRoles: string[];
    modNotificationDM: boolean;
    strictModPermissionCheck: boolean;
    executeWhoisOnJoin: boolean;
    modPublicNotificationWebhook: string;
    modInternalNotificationWebhook: string;
    publishModeratorInfo: boolean;
    preferredLanguage: number;
}

File: invitationtracker/src/modules/database/src/main/java/tracker/database/GuildConfig.java
package tracker.database;

import lombok.AllArgsConstructor;
import lombok.Value;

@Value
@AllArgsConstructor
public class GuildConfig
{
    int id;
    String guildId;
    String modRoles;
    String adminRoles;
    String mutedRoles;
    String internalWebhook;
    String publicWebhook;
    Boolean strictModPermissionCheck;
    Boolean executeWhoisOnJoin;
}
